app/containers/channelPanel.js

```diff
@@ -3,52 +3,36 @@ import { connect } from 'react-redux'
 
 import {
   hideChannelPanel,
-  leaveChannel,
-  moderationAddAdmin,
-  moderationAddMod,
-  moderationBlock,
-  moderationHide,
-  moderationRemoveAdmin,
-  moderationRemoveMod,
-  moderationUnblock,
-  moderationUnhide
+  leaveChannel
 } from '../actions'
 import MemberList from './memberList'
 
 const mapStateToProps = state => ({
   addr: state.currentCabal,
-  cabal: state.cabals[state.currentCabal]
+  currentChannel: state.cabals[state.currentCabal].channel || ''
 })
 
 const mapDispatchToProps = dispatch => ({
   hideChannelPanel: ({ addr }) => dispatch(hideChannelPanel({ addr })),
-  leaveChannel: ({ addr, channel }) => dispatch(leaveChannel({ addr, channel })),
```

Comment:
all of the removed functionalities are in `profilePanel`, and not used here hence removed